example/timeseriesflattener/generate_features_and_write_to_disk.py

```diff
@@ -116,7 +118,21 @@
     flattened_df_ids = flattened_df.df["dw_ek_borger"].unique()
 
     # Version table with current date and time
-    file_prefix = f"psycop_t2d_{time.strftime('%Y_%m_%d_%H_%M')}"
+    # prefix with user name to avoid potential clashes
+    current_user = Path().home().name + "_"
+    file_prefix = current_user + f"psycop_t2d_{time.strftime('%Y_%m_%d_%H_%M')}"
+
+    # Log poetry lock file and file prefix to WandB for reproducibility
+    feature_settings = {
+        "filename": file_prefix,
+        "save_path": SAVE_PATH,
+        "resolve_multiple": RESOLVE_MULTIPLE,
+        "lookbehind_days": LOOKBEHIND_DAYS,
+        "predictor_list": PREDICTOR_LIST,
+    }
```

Comment:
```suggestion
    feature_settings = {
        "filename": file_prefix,
        "save_path": SAVE_PATH,
        "predictor_list": PREDICTOR_LIST,
    }
```

Find it high likelihood that we won't want to share `RESOLVE_MULTIPLE` across all predictors, so might not want to log it by default.

Reply:
Agreed - committing